src/Npgsql/Npgsql.csproj

```diff
@@ -5,10 +5,8 @@
     <Description>Npgsql is the open source .NET data provider for PostgreSQL.</Description>
     <PackageTags>npgsql;postgresql;postgres;ado;ado.net;database;sql</PackageTags>
     <PackageReadmeFile>README.md</PackageReadmeFile>
-    <!-- At this point we target netcoreapp3.1 to avoid taking a dependency on System.Text.Json, which is
-         necessary for older TFMs. -->
-    <TargetFrameworks>netstandard2.0;netstandard2.1;netcoreapp3.1;net5.0;net6.0;net7.0</TargetFrameworks>
```

Comment:
This will have to be documented as a potentially breaking change, right?

Reply:
Yes, probably.
Doing this, means that, as of Npgsql 8, projects targeting .NET Core 3.1 or .NET 5 will no longer be supported directly but rather via .NET Standard 2.1.

Reply:
After reviewing all our #if NETXXX directives I can tell that the most invasive breaking change (loss of a feature that used to work previously) will be the fact that PEM certificates will become unsupported for .NET 5 when it falls back to .NET Standard 2.1.
Other than that we can pretty much avoid any degradation of functionality for the two (already unsupported) .NET (Core) versions we'll be dropping the TFMs for.